src/main/java/seedu/address/logic/commands/UFindCommand.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.logic.commands;
+
+/**
+ * Finds and lists all persons in address book whose name or address contains any of the argument keywords.
+ * Name or address searched depends on prefix in user input.
+ * Keyword matching is case-insensitive.
+ */
+public class UFindCommand {
```

Comment:
Interesting choice to not extend command! I'm curious to know if its for any specific reason.

Reply:
Dear Bhanuka, I hope this message finds you well. The rationale for this decision is that I already have two features (find and find_add) that serve the functionality of searching up contacts based on their name and address respectively. Hence, I will be capitalizing on these features to implement by unified find feature. Thus, my unified find feature does not do the "heavy lifting" of searching up the address book.